eclog/src/main/java/com/ericsson/bss/cassandra/ecaudit/eclog/config/EcLogYamlConfigLoader.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.eclog.config;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ericsson.bss.cassandra.ecaudit.eclog.ToolOptions;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.Constructor;
+import org.yaml.snakeyaml.error.YAMLException;
+
+public final class EcLogYamlConfigLoader
+{
+    private static final EcLogYamlConfig DEFAULT_CONFIG = new EcLogYamlConfig();
+    private static final String FILE_NAME = "ecLog.yaml";
+
+    private EcLogYamlConfigLoader()
+    {
+        // static class
+    }
+
+    public static EcLogYamlConfig load(ToolOptions options)
+    {
+        return getConfigFileLocations(options).stream()
+                                              .filter(Files::isReadable)
+                                              .map(EcLogYamlConfigLoader::loadConfigFromUrl)
+                                              .findFirst()
```

Comment:
Question - Should we print the location of the file actually being used? 
Maybe it could be confusing if there is an ecLog.yaml file in both the working directory AND together with the chronicle files...